src/plugins/advanced_settings/public/management_app/settings_allowlist.ts

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+const GENERAL_SETTINGS = [
+  'csv:quoteValues',
+  'dateFormat:dow',
+  'dateFormat:scaled',
+  'dateFormat:tz',
+  'dateNanosFormat',
+  'defaultIndex',
+  'filters:pinnedByDefault',
+  'format:bytes:defaultPattern',
+  'format:currency:defaultPattern',
+  'format:number:defaultLocale',
+  'format:number:defaultPattern',
+  'format:percent:defaultPattern',
+  'metaFields',
+  'state:storeInSessionStorage',
+  'timepicker:quickRanges',
+  'timepicker:timeDefaults',
+];
+
+const PRESENTATION_LABS_SETTINGS = ['labs:dashboard:deferBelowFold'];
+
+const ACCESSIBILITY_SETTINGS = ['accessibility:disableAnimations'];
+
+const AUTOCOMPLETE_SETTINGS = ['autocomplete:valueSuggestionMethod'];
+
+const BANNER_SETTINGS = [
+  'banners:placement',
+  'banners:textContent',
+  'banners:textColor',
+  'banners:backgroundColor',
+];
+
+const DISCOVER_SETTINGS = ['defaultColumns'];
+
+const NOTIFICATION_SETTINGS = [
+  'notifications:banner',
+  'notifications:lifetime:banner',
+  'notifications:lifetime:error',
+  'notifications:lifetime:info',
+  'notifications:lifetime:warning',
+];
+
+const SEARCH_SETTINGS = ['courier:ignoreFilterIfFieldNotInIndex', 'search:timeout'];
+
+export const ALL_ALLOWLIST_SETTINGS = [
```

Comment:
the plan was to set this stuff from the `serverless` plugin, right?

Reply:
That's correct. I'm only testing it in the original advanced settings plugin.

@clintandrewhall is currently working on extracting the Advanced settings UI components into a package, and once it is done, we will reuse it in the `serverless` plugin so that we register the Advanced settings app there. Then we can call `setAllowlist()` from the `serverless` plugin and put it all together.